en/config/actuators.md

```diff
@@ -591,8 +592,14 @@ If any motors do not turn in the correct direction they must be reversed.
 
 There are several options:
 
-- If the ESCs are configured as [DShot](../peripherals/dshot.md) you can reverse the direction via UI (**Set Spin Direction** buttons).
-  Note that the current direction cannot be queried, so you might have to try both options.
+- If the ESCs are configured as [DShot](../peripherals/dshot.md) you can permanently reverse the direction via UI.
```

Comment:
FYI only @bkueng I added diagram here to make it more obvious how the DShot ESC are set. We can fix if a way is found to improve the UI.